test/test_nn.py

```diff
@@ -5121,7 +5121,7 @@ def test_sparse_default_std(self):
 
                 for col_idx in range(input_tensor.size(1)):
                     column = input_tensor[:, col_idx]
-                    assert column[column == 0].nelement() >= math.ceil(sparsity * cols)
+                    assert column[column == 0].nelement() >= math.ceil(sparsity * cols), "{} : {}, {}".format(column[column == 0].nelement(), sparsity, cols)
```

Comment:
Hi,
I meant to change this to `sparsity * rows`, as in the `nn.init`

Reply:
oh wow... pushing now.